hatt_classifier.py

```diff
@@ -147,12 +147,12 @@ def load_data(train_set,
 
 	if multilabel:
 		mlb = MultiLabelBinarizer()
-		mlb.fit([class_dict.values()])
+		mlb.fit([list(class_dict.values())])
```

Comment:
already a list

Reply:
In Python 3 `.values` is not a list type:

```
$ python3
Python 3.6.9 (default, Apr 18 2020, 01:56:04) 
[GCC 8.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> test = {1: "a", 2: "b"}
>>> type(test.items())
<class 'dict_items'>
```

If `mlb.fit` will deal with the `dict_items` type properly then feel free to add a commit to do what you are suggesting as that will work, I don't know what `MultiLabelBinarizer` does and you have no type stubs which means I can't easily do a type check with mypy so I'm just being conservative here by turning it into a list and matching existing Python2 behavior.